src/timeline/js/mixin_webengine.js

```diff
@@ -39,6 +39,7 @@ function init_mixin() {
     angular.element(document).ready(function () {
       timeline.qt_log("INFO", "Angular Ready");
       $("body").scope().enableQt();
+      $("body").addClass("webengine");
```

Comment:
Adding a custom class to the <body class="webengine">, so we can use it to target the difference backends using CSS.